### ISSUE TITLE
Performance is sluggish on the Measure screen and Graphs screen.

### ISSUE THREAD

**Issue by samreid** [phetsims/energy-skate-park#213]
From #202: Performance is sluggish on the Measure screen and Graphs screen.  Slow/juddery/stuttery much like https://github.com/phetsims/energy-skate-park/issues/206

**Comment by jessegreenberg** [comment]
Performance is fairly jittery on the Intro screen as well, but a bit better on the Playground screen. Indicates it is related to SkaterSamples and display of the skater path, but ill do some profiling.

**Comment by jessegreenberg** [comment]
Recorded time allocation  on a Mac:
![image](https://user-images.githubusercontent.com/6396244/82494782-1f6b3100-9ab8-11ea-81a1-9e537f06d556.png)

A fair amount of time spent painting. `drawDataScatter` stands out here.

**Comment by jessegreenberg** [comment]
`drawDataScatter` is pretty slow because every scatter point gets its own `stroke` and `fill` call, rather than calling `stroke` and `fill` once for every point. But each scatter point can have its own color, opacity, stroke style, and so on, so it is necessary to do so. I also considered optimizing by collecting points with the same stroke/fill style and drawing them once. But I didn't see a performance improvement in the profiler when I reduced the number of times `fill` and `stroke` were called, so I stopped going down this path.

Next I investigated using `layerSplit`, hoping that spliting things into a different layer would help. I tried it on the static background (hoping to prevent redraw of this), the "top layer" of the EnergySkateParkScreenView (hoping to prevent repaint of things that don't move), as well as just the scatter plot. But none of these helped.

Next I looked into using `preventFit`, which finally produced some benefit. With `?showFittedBlockBounds`, I saw a fitted block around the skater update every frame so I added `preventFit: true` to SkaterNode. It resulted in much smother animation and the following:
![image](https://user-images.githubusercontent.com/6396244/82504878-6feb8a00-9aca-11ea-8344-98de59e06e5f.png)

There is less time spent in layout compared to https://github.com/phetsims/energy-skate-park/issues/213#issuecomment-631713635
